# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sysklogd/arch-alpha-ksymmod.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./ksym_mod.c.orig	Sun Oct  8 00:10:58 2000
+++ ./ksym_mod.c	Sun Oct  8 00:12:26 2000
@@ -80,6 +80,8 @@
  *	Changed llseek() to lseek64() in order to skip a libc warning.
  */
 
+/* On Alpha system this header file produces a compiler error */
+#define _ALPHA_CURRENT_H   // disable asm/current.h [2.4.0-test9]
 
 /* Includes. */
 #include <stdlib.h>
